srcup/cli.py

```diff
@@ -34,8 +36,30 @@ def single(
     api_key: str = typer.Option(..., envvar="WD_API_KEY", help="Watchdog API key"),
     owner_username: str = typer.Option('', help="Username of project owner. Ignored when --init is also present"),
     name: str = typer.Option('', help="Project name"),
-    comment: str = typer.Option('', help="Comment for the project")
+    comment: str = typer.Option('', help="Comment for the project"),
+    show_version: bool = typer.Option(False, '--version', '-v', help="Show the version of the app")
 ):
+    if target == 'DEFAULT_ARGUMENT' and not show_version:
+        raise typer.BadParameter("Missing argument 'TARGET'")
+
+    app_version = importlib.metadata.version('srcup')
+
+    if show_version:
+        print(app_version)
+        return
+
+    latest_app_version = asyncio.run(get_latest_app_version())
+    if not latest_app_version:
+        print("Error: Failed to retrieve the latest version of the app")
+        return
```

Comment:
I think we can be more relaxed here about errors. We can just issue a warning and continue the execution normally.

Reply:
This can happen only when no tags have ever been created to the repo or when Github is down. So, ensuring the former, the latter is also a bit unlikely to cause any problems. 

I can relax the constraint if you prefer though, sure.

Reply:
Uploaded the relaxed version